querybook/server/logic/query_review.py

```diff
@@ -27,12 +27,23 @@ def create_query_review(
         session=session,
     )
 
-    # Add reviewers to the query_review assigned reviewers relationship
     for reviewer_id in reviewer_ids:
         reviewer = get_user_by_id(reviewer_id, session=session)
         if reviewer:
+            # Add reviewers to the query_review assigned reviewers relationship
             query_review.assigned_reviewers.append(reviewer)
 
+            # Add reviewer as viewer
+            QueryExecutionViewer.create(
+                fields={
+                    "query_execution_id": query_execution_id,
+                    "uid": reviewer_id,
+                    "created_by": query_execution.uid,  # Original query review creator
+                },
+                commit=False,
+                session=session,
+            )
```

Comment:
check if reviewer has access to the query before assigning them. they might have gotten access if they are already invited to the data doc